pkg/models/devops/devops.go

```diff
@@ -190,7 +187,7 @@ func (d devopsOperator) CreateDevOpsProject(workspace string, project *v1alpha3.
 }
 
 func (d devopsOperator) GetDevOpsProject(workspace string, projectName string) (*v1alpha3.DevOpsProject, error) {
-	return d.ksInformers.Devops().V1alpha3().DevOpsProjects().Lister().Get(projectName)
+	return d.ksclient.DevopsV1alpha3().DevOpsProjects().Get(context.Background(), projectName, metav1.GetOptions{})
```

Comment:
`devopsOperator` already has a `context`. So, please use `d.context` instead of creating a new one `context.Background()`

Reply:
Make sense to me.

Reply:
Actually, it's not true. Function `context#Background()` always returns an initialized, non-nil, empty Context.

Source code of context#Background() as follow:

```go
var (
	background = new(emptyCtx)
	todo       = new(emptyCtx)
)

// Background returns a non-nil, empty Context. It is never canceled, has no
// values, and has no deadline. It is typically used by the main function,
// initialization, and tests, and as the top-level Context for incoming
// requests.
func Background() Context {
	return background
}
```

Reply:
Why do you always need an initialized, non-nil, empty Context?

Reply:
By the way, you now can rebase with the master branch. I fixed the unit test failure from #28